isign/bundle.py

```diff
@@ -144,6 +158,14 @@ def sign(self, signer):
                 appex = signable.Appex(self, appex_exec_path)
                 appex.sign(self, signer)
 
+        # sign all tweak's dylibs
```

Comment:
OK I see why you might need this, but I'd never seen dylibs just in the main directory.  However, this is just copied code from what's right above. Refactor?